frontend/components/Calendar/Month.vue

```diff
@@ -33,7 +33,7 @@ export default {
   props: {
     date: {
       type: String,
-      default: '2021-01',
+      default: '2021-01-01',
```

Comment:
デフォルトをYYYY-mm-dd形式に変更していますが、date propsが入力されていない場合、save()でdefaultの値に-01をつけてしまい、返り値がYYYY-mm-dd-01になってしまいます。
また、そこまで重要ではないと思いますが、デフォルトの値を当日にしてもいいかもしれません。
```suggestion
      default: '2021-01',
```
↓デフォルトを当日の月にする場合↓
```suggestion
      default: new Date().toISOString().substr(0, 7),
```

Reply:
修正しました。